src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(4, 2)).toStrictEqual([2, 2]);
```

Comment:
```suggestion
  expect(splitInteger(4, 2))
    .toStrictEqual([2, 2]);
```

Reply:
Sorry, it's just the way Vadim wrote the code in the lecture

Reply:
OK, no problem.

Reply:
Again - it is just a suggestion for convenience. You can ignore it if you want )